libs/portals/admin/ids-admin/src/components/Client/Translations.tsx

```diff
@@ -39,10 +35,11 @@ const Translations = ({ translations }: TranslationsProps) => {
   return (
     <ContentCard
       title={formatMessage(m.translations)}
-      onSave={(saveOnAllEnvironments) => {
-        console.log('saveOnAllEnvironments: ', saveOnAllEnvironments)
+      onSave={() => {
```

Comment:
Looks like the `onSave` function is redundant, and in the other components. What is the purpose of the function other than control rendering of the 'Save in all environments' checkbox?

Reply:
I added it before I added the loader pattern to the request, I forgot to remove it once the loader pattern was implemented :D